lib/slack/web/client.rb

```diff
@@ -10,7 +10,7 @@ class Client
 
       def initialize(options = {})
         Slack::Web::Config::ATTRIBUTES.each do |key|
-          send("#{key}=", options[key] || Slack::Web.config.send(key))
+          send("#{key}=", options.fetch(key, Slack::Web.config.send(key)))
```

Comment:
This turned out to not be directly relevant, but I noticed that if there was a truthy value in the config that it could not be overridden with a falsey value.